.changeset/eight-cups-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': major
+---
+
+Changes return type of outgoingPayment.create to OutgoingPaymentWithSpentAmounts to reflect correct return type. Consumers should update code to account for new return type as needed.
```

Comment:
Major since it may break consumers. such as here perhaps: https://github.com/interledger/web-monetization-extension/pull/386/files

Reply:
Isn't it technically just adding new optional fields though? i.e. it wouldn't necessarily break current consumers

Reply:
could break at build, no?

Reply:
Just in case, it won't break for extension  (I aliased the import there). So I'm happy with a patch change for this.

Reply:
yeah, you're right. updated to patch